swirl/connectors/connector.py

```diff
@@ -393,6 +396,7 @@ def save_results(self):
                 result.retrieved = result.retrieved + self.retrieved
                 result.time = f'{result.time + (end_time - self.start_time):.1f}'
                 result.json_results = result.json_results + self.processed_results
+                self.cat_results()
                 result.query_processors = query_processors
```

Comment:
I think you want to remove the line above the call to cat_results() since it does that, and more now.